recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+use_pandoc:
+  - yes
```

Comment:
i have fear of yaml turning `yes` and `no` into unexpected things...

Reply:
maybe just

```yaml
with_nbconvert:
- nbconvert-core  # or "-core"
- nbconvert # or ""
```

and reuse them directly?

Reply:
This is much more elegant!

> i have fear of yaml turning yes and no into unexpected things...

The fact my first (local) version failed due to be expecting `yes`/`true` to be turned into booleans proves this point 😆